src/CoreShop/Bundle/ResourceBundle/DependencyInjection/Compiler/StackClassesPass.php

```diff
@@ -55,11 +57,12 @@ public function process(ContainerBuilder $container)
                 if (in_array($interface, class_implements($definition['classes']['interface']))) {
                     $classStack[$alias][] = $definition['classes']['model'];
 
-                    $fullClassName = $definition['classes']['model'];
-                    $class = str_replace('Pimcore\\Model\\DataObject\\', '', $fullClassName);
-                    $class = str_replace('\\', '', $class);
-
-                    $classStackPimcoreClassName[$alias][] = $class;
+                    try {
+                        $reflectionClass = new ReflectionClass($definition['classes']['model']);
```

Comment:
@BlackbitNeueMedien  this can be tricky, the class could not exist yet, since they either weren't installed yet or not created yet.